packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2101,4 +2101,40 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertText('initial value');
   }
+
+  ['@htmlbars a two way binding flows upstream when set during init (#11192)']() {
```

Comment:
No, no, I didn't take it as pedantic, but I wasn't sure (until you clarified) if the issue was with my horrible prose, or the concept being tested.